life_expectancy/cleaning.py

```diff
@@ -38,14 +38,14 @@ def clean_data(_df: pd.DataFrame, region_name: str) -> pd.DataFrame:
     return data
 
 
-def save_data(data) -> None:
+def save_data(data: pd.DataFrame) -> None:
     '''
     Saves the data to a new file.
     '''
     data.to_csv(DIR_PATH / "data" / SAVE_FILE_NAME, index=False)
 
 
-def parse_args():
+def parse_args_pt():
```

Comment:
Question: why was this renamed "_pt"?

Reply:
I thought it could cause a conflict when I patched the method 'parse_args()' since parse_args is also called inside the method. I reverted it though as it is unnecessary.